build-images/github-webhook-receiver/go.sum

```diff
@@ -28,39 +28,135 @@ cloud.google.com/go v0.93.3/go.mod h1:8utlLll2EF5XMAV15woO4lSbWQlk8rer9aLOfLh7+Y
 cloud.google.com/go v0.94.1/go.mod h1:qAlAugsXlC+JWO+Bke5vCtc9ONxjQT3drlTTnAplMW4=
 cloud.google.com/go v0.97.0/go.mod h1:GF7l59pYBVlXQIBLx3a761cZ41F9bBH3JUlihCt2Udc=
 cloud.google.com/go v0.99.0/go.mod h1:w0Xx2nLzqWJPuozYQX+hFfCSI8WioryfRDzkoI/Y2ZA=
+cloud.google.com/go v0.100.1/go.mod h1:fs4QogzfH5n2pBXBP9vRiU+eCny7lD2vmFZy79Iuw1U=
```

Comment:
This file shouldn't have been changed by this PR, please revert it to what it should have been

Reply:
reverted to commit https://github.com/galasa-dev/automation/commit/482f39a54d4a2a66e7f55f963d4b21bc2e4713a3
git checkout 482f39a -- build-images/github-webhook-receiver/go.sum

Reply:
reverted to commit https://github.com/galasa-dev/automation/commit/482f39a54d4a2a66e7f55f963d4b21bc2e4713a3
which removed the 706 additions